x-pack/plugins/security_solution/public/timelines/components/flyout/header/index.tsx

```diff
@@ -137,14 +132,15 @@ const FlyoutHeaderPanelComponent: React.FC<FlyoutHeaderPanelProps> = ({ timeline
               <AddToFavoritesButton timelineId={timelineId} compact />
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
-              <ActiveTimelinesContainer grow={false}>
-                <ActiveTimelines
-                  timelineId={timelineId}
-                  timelineType={timelineType}
-                  timelineTitle={title}
-                  isOpen={show}
-                />
-              </ActiveTimelinesContainer>
+              <EuiText
+                grow={false}
+                data-test-subj="timeline-title"
+                css={css`
+                  white-space: nowrap;
+                `}
```

Comment:
Does `css` always return the same or do we expect this to lead to rerenders?

Reply:
crap I forgot to remove this one... 😭
Damn I hate this thing so much 😆 
Let me extract this into a constant... 😞

Reply:
actually I think i remember extracting it but the change wasn't applied... I'll ask the team during standup!

Reply:
fixed [here](https://github.com/elastic/kibana/pull/173886/commits/e8449dd70f0d39641aa8ea4fbcae98ef285456ce)